modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -144,14 +157,34 @@ export default class Combobox extends React.Component<ComboboxProps, ComboboxSta
     }
   }
 
+  getAutoCompleteItems = () => {
+    let items: React.ReactElement<ComboBoxMenuItemProps>[] = []
+    console.log(this.props.autocompleteItemGroup)
+    if (this.props.autocompleteItemGroup && this.props.autocompleteItemGroup.length) {
+      items = this.props.autocompleteItemGroup.reduce((total, groupItem) => {
+        console.log(groupItem)
+        return [
+          ...total,
+          ...groupItem.items,
+        ]
+      }, items)
+    } else if (this.props.autocompleteItems && this.props.autocompleteItems.length) {
+      items = this.props.autocompleteItems
+    }
+    return items
+  }
```

Comment:
I'm not a fan of `let`. I believe the following could be refactored and types removed from the implementation:

```suggestion
  getAutoCompleteItems = (): React.ReactElement<ComboBoxMenuItemProps>[] => {
    if (this.props.autocompleteItemGroup && this.props.autocompleteItemGroup.length) {
      return flatten(this.props.autocompleteItemGroup.map(group => group.items))
    }
    return this.props.autocompleteItems || []
  }
```

`flatten` comes from:
```ts
import flatten from 'lodash/flatten'
```

[TS Playground](https://www.typescriptlang.org/play/?ssl=19&ssc=6&pln=16&pc=6#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wChSB6AKjgDkJ4B3aAa2ADsBzODuMAGxQBPTjgCu7ACYA6WXDEBnJHBRxJSNIKI92CmKkmlQkWHEyCY+9mZx58-CJJQKAFhXMpLSdmSoVS6poo2pgSGMAQ1h5e7AA8ACoAfAAUwPogCgBccACCUFDCCXAAPnDxANoAuokAlNkVleQwQmDKAMK4AEYQAEIQAB4Ast5iAJLpAAo4YApwALxwAN4AvqTNrXAdIN19QyPjSCAA4uJg80ukcDzpWYjEMNLI6DAAovyH3jCxWzsDw+xjSbTBSJKqkVZrFrKeIuYCzBaLMDA7KLFRiGAQNC4ARIfQHY6nAD82R+vT++3SJwgYjAVQANGiMVjwO88Tdsk8MI97m8PuwvqTdv9AYcphAZqDKssIaF2OFInBOLicuiIFscWzDrNkjBYbcYXC6ndntznryQJ9vl0yXsAfixRKqhcrsBMHAdXrpEjxQppChVcyNUh8VSaXAAGThuC6uFe4F+gPY1nBymnaTvLi6mrOq5wIgwMRQKIWKwe2PemYJplJ3Epw6hsDSEAoMDJUTUs5zRKKtNpLU1GqXOCrK75wvWGO+iu+-3Vlm1-GzYqlMEysIwCLWJUwFUY9XJxcAeX4kndk-qeuziyHV1Z1y1HPupow5stgvJdqBPsl5zBubgWK6BA7zphAnBllO8azpiNaagSHaDv+rpnp605VjB85wQ2EZRpOcY+uhgYHqmHbpt4nBZjmSE3OceFodBRELiRNLSEQkhiGgSDJDqDAoPwDLtjS+LZl2VH-gBkQKMBSCgeBglgMJN7iWORZwOUSniXAsjSBiMB8XSGnidp8n4tIfYZAZmlXI0mnLAy5kKIhubLHASD8EoPBuhB+GVgxsF1hkOHRqhUGJphAW+hmFEuFehkObRIUEX54WLkpI65ip1gOUOyxAA)